examples/yum_collection_migration_spec.yml

```diff
@@ -0,0 +1,40 @@
+# a draft of a spec file to migrate content from core to a collection, for test purposes
+
+yum_collection:
+  action:
+    - yum.py
+  module_utils:
+    - yumdnf.py
+  modules:
+    - packaging/os/yum.py
+  units:
+    - test_yum.py
+  targets:
```

Comment:
should this be under some `test/` subsection?

Reply:
`test` is already taken by test plugins.

Reply:
It's just weird to mix it up. Maybe `test_targets`/`test_units`? or `integration_tests`/`unit_tests`?

Reply:
Yeah, I suspect bikeshedding needs to happen. Also we might end up doing 'auto find' on those?

Reply:
Not sure we could do this reliably.

Reply:
OTOH if ansible-test already has such discovery, it might work.

Reply:
not sure we need to specify units/integration as much as 'discover'

Reply:
Maybe let's just omit integrations/units for now.

Reply:
fine, doing it incrementally is better imho, also might require changing schema of spec file, i did not really account for tests being specified, thoguht they would always be 'automated'

Reply:
+1, removed